OpenDreamRuntime/Objects/Types/DreamObjectWorld.cs

```diff
@@ -314,4 +317,20 @@ private void SetFps(DreamValue fps) {
 
         Fps = (int)Math.Round(fpsValue);
     }
+
+    private void SetTicklag(DreamValue value) {
+        if (!value.TryGetValueAsFloat(out var tickLag))
+            tickLag = 1; // An invalid tick_lag gets turned into 1
+
+        TickLag = tickLag;
+    }
+
+    private void SetSleepOffline(DreamValue sleepOffline) {
+        if (sleepOffline.TryGetValueAsInteger(out var value) && value == 1) {
```

Comment:
Truthy values are treated as true here.
```suggestion
        if (sleepOffline.IsTruthy()) {
```